DS13/code/modules/EMH/EMH.dm

```diff
@@ -0,0 +1,291 @@
+GLOBAL_LIST_INIT(EMH_blacklist, list())
+
+//EMITTER
+
+/obj/machinery/emh_emitter
+	name = "Emergency medical hologram emitter"
+	desc = "Simply click this device and an emergency medical hologram will be summoned. Swipe it with a captain level ID to terminate the current EMH if it's misbehaving."
+	icon = 'DS13/icons/obj/decor/wall_decor.dmi'
+	icon_state = "emh-off"
+	anchored = TRUE
+	density = FALSE
+	pixel_y = 32 //Put on the tile below a wall etc etc modular fun time mapping mayhem
+	var/range = 10 //10 tiles? Pretty damn generous.
+	var/mob/living/carbon/human/species/holographic/emh
+	req_access = list(ACCESS_CAPTAIN)
+
+/obj/machinery/emh_emitter/proc/EMH_present()
+	var/mob/living/carbon/human/species/holographic/S = locate(/mob/living/carbon/human/species/holographic) in GLOB.alive_mob_list
+	if(istype(S) && !QDELETED(S))
+		return TRUE
+	else
+		return FALSE
+
+/obj/machinery/emh_emitter/examine(mob/user)
+	. = ..()
+	if(EMH_present())
+		if(emh.client)
+			to_chat(user, "<i>The panel shows that there is an active EMH on the network.</i>")
+		else
+			to_chat(user, "<i>The panel shows that there is an EMH on the network, but it is not responding (braindead). Click the panel to summon a new one.</i>")
+
+/mob/verb/editemhblacklist()
```

Comment:
This should be a client or holder proc, not a mob verb

Reply:
It must also be added to https://github.com/DDMers/Deep-Space-13/blob/master/code/modules/admin/admin_verbs.dm#L24